src/pages/components/NavContainer/NavContainer.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from "react";
+import Nav from "./Nav/Nav";
+import MyCurator from "./MyCurator/MyCurator";
+import SearchBox from "./SearchBox/SearchBox";
+import "./NavContainer.scss";
+
+class NavContainer extends Component {
+  constructor() {
+    super();
+    this.state = {
+      userName: null,
+      isMyCurator: false,
+      isSearchBox: false
+    };
+  }
+
+  componentDidMount() {
+    const userName = localStorage.getItem("name");
+    this.setState({ userName: userName });
```

Comment:
Key와 value값이 같을 경우 아래와 같이 작성하실 수 있습니다. 
```suggestion
    this.setState({ userName });
```

Reply:
코드보면서 불필요한거 있으면 다같이 수정할게요! 해당 파일은 수정했습니다!